internal/cli/kraft/cloud/deploy/deploy.go

```diff
@@ -63,7 +63,7 @@ type DeployOptions struct {
 	ScaleToZero            bool                      `local:"true" long:"scale-to-zero" short:"0" usage:"Scale the instance to zero after deployment"`
 	ServiceGroupNameOrUUID string                    `long:"service-group" short:"g" usage:"Attach the new deployment to an existing service group"`
 	Strategy               packmanager.MergeStrategy `noattribute:"true"`
-	SubDomain              string                    `local:"true" long:"subdomain" short:"s" usage:"Set the name to use when provisioning a subdomain"`
+	SubDomain              []string                  `local:"true" long:"subdomain" short:"s" usage:"Set the names to use when provisioning subdomains"`
```

Comment:
```suggestion
	SubDomains             []string                  `local:"true" long:"subdomain" short:"s" usage:"Set the names to use when provisioning subdomains"`
```

We should also probably indicate "repeat for multiple values" (or is it comma-separated?) in the usage string.

Reply:
you repeat the flag for multiple values afaik `-s ... -s ...`

Reply:
Then we should have _some way_ to communicate this to the user via `--help`, because the flag's usage alone doesn't indicate how to pass multiple values, and each CLI tool has their own way of doing that.

Reply:
I see now that KraftKit indicates the type `strings` for such flag:
```
-e, --env strings            Environmental variables
```

We cannot possibly write "repeat for multiple values" in every []string flag, so maybe an example that uses multiple domain values is enough. In this case, feel free to ignore my comment.